.travis.yml

```diff
@@ -8,6 +8,10 @@ matrix:
       env: TOXENV=py27
     - python: "2.7"
       env: TOXENV=purepy27
+      before_install:
+        - sudo apt-get remove -y --allow-remove-essential gnupg gnupg2
+      after_script:
+        - sudo apt-get install -y gnupg gnupg2
```

Comment:
Out of curiosity, what's idea of restoring this? Isn't the worker just dumped after running? Is there caching? Anyways, removing gnupg [removes apt](https://travis-ci.org/secure-systems-lab/securesystemslib/jobs/647078797#L258) too, so  the after script [command fails](https://travis-ci.org/secure-systems-lab/securesystemslib/jobs/647078797#L334).

Reply:
Whoops, my bad. I wasn't sure whether there was a new host instance per matrix entry and meant to check before pushing the PR.

I'll remove this.